src/opera_utils/_utils.py

```diff
@@ -153,3 +157,94 @@ def scratch_directory(
 
     if delete and not dir_already_existed:
         shutil.rmtree(scratchdir)
+
+
+def get_snwe(epsg: int, bounds: Bbox) -> Bbox:
+    """Convert bounds from (west, south, east, north) (WSEN) to SNWE in lat/lon.
+
+    This box format is used by the `RAiDER` library for its area of interest tracking:
+    https://github.com/dbekaert/RAiDER/blob/38eab3969ac762bc59502d7eb482fd73d6a0deef/tools/RAiDER/llreader.py#L30
+
+    Parameters
+    ----------
+    epsg : int
+        EPSG code.
+    bounds : tuple[float, float, float, float]
+        Bounds in WSEN format.
+
+    Returns
+    -------
+    tuple[float, float, float, float]
+        Bounds in SNWE (lat/lon) format.
+    """
+    if epsg != 4326:
+        snwe = reproject_bounds(bounds, epsg, 4326)
+
+    else:
+        snwe = (bounds[1], bounds[3], bounds[0], bounds[2])
```

Comment:
```suggestion
    if epsg != 4326:
        snwe = reproject_bounds(bounds, epsg, 4326)

    snwe = (bounds[1], bounds[3], bounds[0], bounds[2])
```
I think right now, if you need to reproject, it'll make reproject `bounds`, but then leave it in `WSEN` order instead of `SNWE`


it'll be good to add one test, `test_get_snwe`, to the `tests/test_utils.py` module where we check the no-reproject case with something like

```python
def test_get_snwe():
    bounds = (-110, 32, -109, 33)
    assert get_snwe(4326, bounds) == (32, 33, -110, -109)
```

Reply:
hmm shouldn't this be like:
`bounds = reproject_bounds(bounds, epsg, 4326)`

Reply:
whoops yes you are right.

(edited my suggestion)